tests/unit/resource/ledger/query_collection_resource_ledger_service_test.go

```diff
@@ -0,0 +1,50 @@
+//go:build unit
+
+package ledger
+
+import (
+	resourceTypes "github.com/cheqd/cheqd-node/api/v2/cheqd/resource/v2"
+	testconstants "github.com/cheqd/did-resolver/tests/constants"
+	utils "github.com/cheqd/did-resolver/tests/unit"
+	"github.com/cheqd/did-resolver/types"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+type queryCollectionResourcesTestCase struct {
+	did                string
+	expectedCollection []*resourceTypes.Metadata
+	expectedError      *types.IdentityError
+}
+
+var _ = DescribeTable("Test something", func(testCase queryCollectionResourcesTestCase) {
```

Comment:
Let's rename it to something like "Test query resource collection for ledger service"

Reply:
Done! ✅